reference/library/src/webapp/editor/ckextraplugins/wordcount/plugin.js

```diff
@@ -82,8 +87,8 @@
 										var editor = evt.editor;
 										space = getSpaceElement(editor);
 										//use getSnapshot because WIRIS is listening for getData. It looks like getData(true) actually doesn't return anything! Hmm!
-										editordata = editor.getSnapshot();
-										space.setHtml( editor.lang.wordcount.WordCountTxt + " : " + getWordCount(editordata) );
+									editordata = editor.getSnapshot();
+									space.setHtml( editor.lang.wordcount.CharCountTxt + " : " + getCharCount(editordata) + " " + editor.lang.wordcount.WordCountTxt + " : " + getWordCount(editordata) );
 									}

```

Comment:
Sure, the formatting in this file is all over the place, not sure why this bumped back a tab though.